src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -73,19 +68,11 @@ public sealed class SessionImpl : AbstractSessionImpl, IEventSource, ISerializab
 		private readonly StatefulPersistenceContext persistenceContext;
 
 		[NonSerialized]
-		private readonly SessionImpl rootSession;
-
-		[NonSerialized]
-		ISession _childSession;
-
-		[NonSerialized]
-		private readonly bool flushBeforeCompletionEnabled;
-		[NonSerialized]
 		private readonly bool autoCloseSessionEnabled;
 		[NonSerialized]
-		private readonly bool ignoreExceptionBeforeTransactionCompletion;
-		[NonSerialized]
 		private readonly ConnectionReleaseMode connectionReleaseMode;
+		[NonSerialized]
+		private readonly bool _transactionCoordinatorShared;
```

Comment:
NH addition, I expect this to be no more needed once transaction handling will be ported.